server/session/session.go

```diff
@@ -33,10 +36,23 @@ func NewServer(namespace string, kubeclientset kubernetes.Interface, appclientse
 const (
 	invalidLoginError  = "Invalid username or password"
 	blankPasswordError = "Blank passwords are not allowed"
+	authCookieName     = "com.intuit.argocd.auth.token"
```

Comment:
No intuit stuff, please. If you need an arbitrary domain, you can use argoproj.io. Can a cookie look like argocd.argoproj.io/auth-token (which follows our meta-data convention for kubernetes resources)